overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -37556,19 +37641,19 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "I\u0027m running out of MCU puns.\n\nThe §aAngel Ring§r is actually slower than a jetpack, but it grants creative flight and requires no fuel (except a bit of GP).\n\nUp to you if you want to use it over a jetpack for day to day stuff, but it\u0027s an ingredient in the next jetpack, so if you want that you\u0027ll need to make one eventually.",
+          "desc:8": "The §6Angel Ring§r is actually slower than most jetpacks. However, it grants §acreative flight§r with no gradual drop and requires no traditional fuel. \n\nIt reserves §b32 GP§r while active and will deactivate if you don\u0027t have enough. Double press the jump key (i.e spacebar) to activate. Your Angel Ring can be active at the same time as a normal jetpack, and the speeds §astack§r.",
```

Comment:
`No Gradual Drop` confusing. Maybe `perfectly stable flight`, or just omit that (creative flight implies stable)